config/locales/en.yml

```diff
@@ -57,16 +57,43 @@ en:
   devise:
     password:
       new:
+        new_tab_title: 'Forgot password?'
         new_title: 'Forgot your password?'
-        email_label: 'Your Email: '
         email_placeholder: 'Enter email'
         submit_forgot_password: 'Send me reset password instructions'
       edit:
+        edit_tab_title: 'Change password'
         edit_title: 'Change your password'
-        password_label: 'New password'
-        confirm_password_label: 'Confirm your new password'
+        password_placeholder: 'New password'
+        confirm_password_placeholder: 'Confirm your new password'
         submit_change_password: 'Change password'
 
+    registrations:
+      new:
+        tab_title: 'Sign up'
+        registration_title: 'Register Page'
+        f_name_placeholder: 'First name'
+        l_name_placeholder: 'Last name'
+        email_placeholder: 'Email'
+        sex:
```

Comment:
you don't need this tranlations, they at least exists in this file,

Reply:
Thanks, fixed!